js/base.js

```diff
@@ -89,6 +89,18 @@ export class API {
         await this.consumer.consume(topic, { callback: callback, ...options });
     }
 
+    /**
+     * Lists the available topics.
+     *
+     * @param {Function|Object} options Object that includes configuration variables.

```

Comment:
Unless optionals are essential for what you need to do let's avoid them, as it brings extra complexity. It's not only an extra argument, it's writing the types for it and making sure the options aren't kafka-specific.